code/script.js

```diff
@@ -224,29 +228,37 @@ const generateBoard = () => {
 
 // Randomly select a person from the characters array and set as the value of the variable called secret
 const setSecret = () => {
-  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)]
-}
+  secret = charactersInPlay[Math.floor(Math.random() * charactersInPlay.length)];
+};
 
 // This function to start (and restart) the game
 const start = () => {
   // Here we're setting charactersInPlay array to be all the characters to start with
   charactersInPlay = CHARACTERS
   // What else should happen when we start the game?
+  generateBoard();  // invokes the function show the board
+  setSecret();      // invokes the function that sets a secret person
+  selectQuestion(); // invokes the function to be able to select questions
+
+  board.style.display = "flex"      // adding flexbox to this id 
```

Comment:
Nicely solved with the "board=flex" and "winOrlose=none "in this section. I had to create a new const named "Playagain" to make it work, this is a better solution!